diff --git a/configuration/sconsConfigFile.py b/configuration/sconsConfigFile.py
index e79152d..120bded 100755
--- a/configuration/sconsConfigFile.py
+++ b/configuration/sconsConfigFile.py
@@ -141,7 +141,7 @@ def setupSunOs(dict):
         dict['LINKFLAGS'] = initCompilerFlags('LINKFLAGS',LINKER_OPTIONS,dict)
         dict['FORTRANFLAGS'] = initCompilerFlags('FORTRANFLAGS',COMPILER_OPTIONS,dict)
         dict['CCFLAGS'] = initCompilerFlags('CCFLAGS',COMPILER_OPTIONS,dict)
-        dict['CCFLAGS'] = mergeLists(dict['CCFLAGS'],GCCFLAGS)
+        dict['CCFLAGS'] = mergeLists(dict['CCFLAGS'],GCCFLAGS + ['-fuse-ld=lld'])
 
         if os.path.basename(dict['FORTRAN']).count('gfortran'):
 
