From d057ff42e4b9294159b543425b8712e58918c588 Mon Sep 17 00:00:00 2001
From: Bryan Marfito <bjmarfito@yahoo.com>
Date: Wed, 18 Jan 2023 22:12:55 +0800
Subject: [PATCH] Corrected the year on the release_history.py of 2.6.2

---
 release_history.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/release_history.py b/release_history.py
index 9c97ef71..0d650129 100755
--- a/release_history.py
+++ b/release_history.py
@@ -68,7 +68,7 @@
             Tag('2.5.3', '', '20210823'),
             Tag('2.6.0', '', '20220214'),
             Tag('2.6.1', '', '20220811'),
-            Tag('2.6.2', '', '20220117'),
+            Tag('2.6.2', '', '20230117'),
 )
 
 
